lib/aws-record/record/item_operations.rb

```diff
@@ -106,6 +106,14 @@ def _invalid_record?(opts)
         end
       end
 
+      def validation_error_message
```

Comment:
Just for consistency's sake (something I'm working on for new changes), please prefix with `_`